utils/json_profiles_merger_lib.py

```diff
@@ -104,26 +108,30 @@ def _accumulate_event_duration(event_list, accum_dict, only_phases=False):
           'cat': 'build phase marker',
           'dur_list': []
       }
-    current_phase_duration_millis = (next_ts - ts) / 1000
+    current_phase_duration_millis = (next_ts - ts) / 1000 # Convert from ns to microseconds
```

Comment:
`ts` is btw already in microseconds. We're converting from microseconds to milliseconds here.

Reply:
Well spotted. I've corrected this throughout.